src/components/SecretsTable/SecretsTable.test.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { fireEvent, render } from 'react-testing-library';
+import SecretsTable from './SecretsTable';
+
+it('SecretsTable renders with no secrets', () => {
+  const props = {
+    initialRows: [
+      {
```

Comment:
`initialRows` should probably just be empty but let's revisit this one later, we may take a different approach for empty states across the experience